audio/out/ao_oss.c

```diff
@@ -48,12 +48,15 @@
 #define AFMT_AC3 -1
 #endif
 
+#ifndef PATH_DEV_DSP
 #define PATH_DEV_DSP "/dev/dsp"
+#endif
+#ifndef PATH_DEV_MIXER
```

Comment:
when is this already defined?
it also doesn't seem to have anything to do with the commit title

Reply:
ok, removed.